From bd742bab7db7642c73b7921c33636d71946fe81a Mon Sep 17 00:00:00 2001
From: Chris Williamson <chris@chrisaw.io>
Date: Wed, 18 Feb 2026 15:10:04 +0000
Subject: [PATCH 8/8] Remove mtectrl and misctrl from GSI builds

These are Pixel-specific tools for managing MTE (Memory Tagging
Extension) boot control and misc partition flags. On GSI builds
targeting non-Pixel devices they serve no purpose and write to
the misc partition at boot and shutdown which can interfere with
the vendor bootloader TEE initialization, potentially breaking
biometric authentication and other TEE-dependent features.
---
 target/product/base_system.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index 35a3c2f..3cd1804 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -229,8 +229,6 @@ PRODUCT_PACKAGES += \
     mke2fs \
     mkfs.erofs \
     monkey \
-    misctrl \
-    mtectrl \
     ndc \
     netd \
     NetworkStack \
-- 
2.52.0

